inference-engine/thirdparty/clDNN/CMakeLists.txt

```diff
@@ -80,11 +80,21 @@ set(CLDNN_BUILD__PROJ_LABEL__clDNN "clDNN")
 
 # ===================================== Include/Link directories =======================================
 
+if(CLDNN_THREADING MATCHES "TBB")
 include_directories(
     ${CLDNN_UTILS__RAPIDJSON_INCDIRS}
     "${CLDNN__CODEGEN_INCDIR}"
     "${CLDNN__API_DIR}"
+    "${IE_MAIN_SOURCE_DIR}/src/inference_engine/src/threading/"
+    "${IE_MAIN_SOURCE_DIR}/src/inference_engine/include/ie/"
```

Comment:
just separate these as a list, set the lists' contents w.r.t the THREADING in separate lines,  and then here include as a list variable..?

Reply:
Fixed it